### PR TITLE
CODE REVIEW REQUEST ON REACT BOOKSTORE PROJECT: MILESTONE-5-STYLING

### PR DESCRIPTION
**Pull request by Moreneecoder**
In this project, I wrote CSS styling for the UI components to conform with the Zeplin design prototype as provided by Microverse. The styled-components are: 
* Navbar component
* BooksList component
* BooksForm component
* Book component
Each component has its own CSS confined to a separate `.CSS` file for easy maintenance.

### CODE REVIEW

File: src/stylesheets/Book.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.Book {
+  display: flex;
```
Comment (rindrajosia) [inline src/stylesheets/Book.css]: - [x] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule.

File: src/stylesheets/Book.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  height: 4.25rem;
+  border: solid 5px #e8e8e8;
+  border-radius: 50%;
+  border-top-color: #4386bf;
+  border-right-color: #4386bf;
+}
+
+.progess-details {
+  flex: 15%;
+  display: flex;
```
Comment (rindrajosia) [inline src/stylesheets/Book.css]: - [x] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi , 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:

## To highlight:

- Your APP works correctly. :+1: 

## Required Changes ♻️

- [x] Styling should be **EXACTLY** the same as the one provided in the designs ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/))

- (Original design)


- (Your APP)


- [x] The text in the Submit button should be: ADD BOOK
- [x] The "UPDATE PROGRESS" button should have a border-radius like your Submit button
- [x] Please, make sure that you use the same font family, font size, layout, font color, font-weight, etc... as the original design. (for example, the book title should be bold, the category in the select option should be grey, the background of the page should be white, ......)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rukundoeric — APPROVED** [review]
Hi @Moreneecoder,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**